### DISCUSSION TITLE
Is it possible to run 2 domains on vernemq?

### DISCUSSION THREAD

**Discussion by Eldho1416** [vernemq/vernemq#2069] [Q&A] [answered]
Hi, 

Is it possible to add 2 different ssl certificates in Vernemq

**Comment by ioolkos** [comment, marked as answer]
You can currently configure TLS listeners on a per-listener basis. That is, you an configure 2 different TLS listeners, each using its server certificate.
It currently does not seem possible to configure multiple server cert/key pairs for the same listener endpoint (the same IP:port). I think here's some related info over at RabbitMQ:
https://github.com/rabbitmq/rabbitmq-server/issues/2060

OTP 25 has added the capability to do that, though, if I understand correctly.

**Reply by Eldho1416** [reply]
Great Thanks
